airbyte-workers/README.md

```diff
@@ -1,6 +1,10 @@
-# Temporal Development
+# airbyte-workers
 
-## Versioning
+This module contains the logic for how Jobs are executed. Jobs are executed using a tool called Temporal.
```

Comment:
Soon: This module contains a temporal workflow which is responsible to make sure that the job are scheduled base on their configuration and update the job status after it being run.

A lot of the logic of The scheduler and the job submitter will be moving there.

Reply:
makes sense! want to just update it when you merge it?